integration-tests/tests/run-notebook.js

```diff
@@ -14,37 +14,19 @@ const testRunNotebookFn = _.flow(
   await click(page, clickable({ textContains: 'View Workspaces' }))
   await signIntoTerra(page, token)
   await dismissNotifications(page)
-  await findElement(page, clickable({ textContains: workspaceName }))
   await waitForNoSpinners(page)
-  await click(page, clickable({ textContains: workspaceName }))
-  await delay(5000)
+
+  await waitForNoSpinnersAfterAction(page, () => click(page, clickable({ textContains: workspaceName })))
```

Comment:
I wonder if something more along the lines of `doThenSpinners` is better? I feel like `doAfterSpinners`reads better, but that's obviously putting the spinner wait on the next thing, so maybe just `await noSpinnersAfter(page, action)`?

Reply:
+1 on `await noSpinnersAfter(page, action)`

Reply:
@kyuksel which signature do you prefer?

`await noSpinnersAfter(page, action)`

`await noSpinnersAfter(page, { action: foo})`

Reply:
I think I'd prefer `await noSpinnersAfter(page, { action: foo})` because the call site would read more clearly; e.g.

```javascript
await noSpinnersAfter(page, { action: () => click(page, clickable({ text: 'Create Notebook' }))})
```

vs.

```javascript
await noSpinnersAfter(page, () => click(page, clickable({ text: 'Create Notebook' })))
```

but I don't feel strongly.